modules/azure/aks.go

```diff
@@ -0,0 +1,72 @@
+package azure
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/services/containerservice/mgmt/2019-11-01/containerservice"
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	"github.com/gruntwork-io/terratest/modules/logger"
+	"github.com/gruntwork-io/terratest/modules/retry"
+)
+
+// GetManagedClustersClient is a helper function that will setup an Azure ManagedClusters client on your behalf
+func GetManagedClustersClient(subscriptionID string) (*containerservice.ManagedClustersClient, error) {
+	// Validate Azure subscription ID
+	subscriptionID, err := getTargetAzureSubscription(subscriptionID)
+	if err != nil {
+		return nil, err
+	}
+
+	managedServicesClient := containerservice.NewManagedClustersClient(subscriptionID)
+	authorizer, err := NewAuthorizer()
+
+	if err != nil {
+		return nil, err
+	}
+
+	managedServicesClient.Authorizer = *authorizer
+
+	return &managedServicesClient, nil
+}
+
+// GetManagedCluster will return ContainerService
```

Comment:
This looks like it is returning a ManagedCluster object, and not ContainerService?

Reply:
Fixed. Done.